### PR TITLE
Updated README practice

### PR DESCRIPTION
**Pull request by chithihuynh**
Added Chi to project and inserted 2 new lines.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,7 +1,9 @@
 Git Test Project
 Shawn Meier
+Chi Huynh
```
Comment (ftc) [inline README]: I think you should put your middle name in here too.

**Review by ftc — CHANGES_REQUESTED** [review]
One tiny suggestion :-)

File: README (hunk 1 above)
Comment (chithihuynh) [reply README]: Done.

**Comment by ftc** [comment]
Resolves #2

**Pull request merged**